### ISSUE TITLE
Error `message check: key "extra": field was not used: parenthesized` for v2.6.0

### ISSUE THREAD

**Issue by zurk** [bblfsh/javascript-driver#67]
There is a file that is parsable with `v1.2.0` driver but not with `v2.6.0`
Full error mesage 
```
Errors:
	- transform failed: received 2 errors:
	check: key "extra": field was not used: parenthesized
	check: key "extra": field was not used: parenthesized
```

How to reproduce 
```bash
wget https://raw.githubusercontent.com/hakimel/reveal.js/0b3e7839ebf4ed8b6c180aca0abafa28c67aee6d/js/reveal.js
```

v2.6.0 javascript-driver
```

docker run -d --rm --name bblfshd_new_driver --privileged bblfsh/bblfshd\:v2.11.0 --log-level DEBUG
docker exec bblfshd_new_driver bblfshctl driver install javascript docker://bblfsh/javascript-driver:v2.6.0
docker cp reveal.js bblfshd_new_driver:reveal.js
docker exec -it bblfshd_new_driver bblfshctl parse reveal.js
```

v1.2.0 javascript-driver (the same script, just install another driver and give different name)
```
docker run -d --rm --name bblfshd_old_driver --privileged bblfsh/bblfshd\:v2.11.0 --log-level DEBUG
docker exec bblfshd_old_driver bblfshctl driver install javascript docker://bblfsh/javascript-driver:v1.2.0
docker cp reveal.js bblfshd_old_driver:reveal.js
docker exec -it bblfshd_old_driver bblfshctl parse reveal.js
```

Last command prints UAST. Be careful, it is huge.

**Comment by bzz** [comment]
Thank you for the report! This has been already fixed by #57 and going to be included in upcoming v2.7.0

Going to keep it open, until you could verify it

**Comment by zurk** [comment]
yes, it works for v2.7.0
thanks!
